docs/tutorials/search-v3.md

```diff
@@ -0,0 +1,31 @@
+# Amundsen `/v2/search`
```

Comment:
I feel like this needs a `tl;dr` and, below that, a "to configure such and such" section.